extensions/label/json-schema/schema.json

```diff
@@ -1,119 +1,135 @@
       "properties": {
-        "type": "object",
-        "required": [
-          "label:properties",
-          "label:classes",
-          "label:description"
-        ],
-        "properties": {
-          "label:properties": {
-            "title": "Property",
-            "type": "array",
-            "items": {
-              "type": "string"
+        "label:properties": {
+          "title": "Property",
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "type": "null"
             }
-          },
-          "label:classes": {
-            "title": "Classes",
-            "type": "array",
-            "items": {
-              "type": "object",
-              "required": [
-                "name",
-                "classes"
-              ],
-              "properties": {
-                "name": {
-                  "title": "Name",
-                  "type": "string"
-                },
-                "classes": {
-                  "title": "Classes",
-                  "type": "array",
-                  "items": {
-                    "type": "string"
+          ]
+        },
+        "label:classes": {
+          "title": "Classes",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [
+              "name",
+              "classes"
+            ],
+            "properties": {
+              "name": {
+                "title": "Name",
+                "type": "string"
+              },
+              "classes": {
+                "title": "Classes",
+                "oneOf": [
+                  {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  {
+                    "type": "array",
+                    "items": {
+                      "type": "number"
+                    }
                   }
-                }
+                ]
               }
             }
-          },
-          "label:description": {
-            "title": "Description",
+          }
+        },
+        "label:description": {
+          "title": "Description",
+          "type": "string"
+        },
+        "label:type": {
+          "title": "Type",
+          "type": "string",
+          "enum": [
+            "raster",
+            "vector"
+          ]
+        },
+        "label:tasks": {
+          "title": "Task",
+          "type": "array",
+          "items": {
             "type": "string"
-          },
-          "label:type": {
-            "title": "Type",
+          }
+        },
+        "label:methods": {
+          "title": "Method",
+          "type": "array",
+          "items": {
             "type": "string",
             "enum": [
-              "raster",
-              "vector"
+              "automated",
```

Comment:
This doesn't follow the doc (highlighted text by me):

> Recommended to be a subset of 'automated' or 'manual', but **may be an arbitrary value.**

Reply:
So I think the whole enum just needs to be removed then.